### DISCUSSION TITLE
ipad and mac have different behaviour when defining input-fields

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#133] [Q&A]
Hi Renaud,

On my Mac I can give an input-field the  type fomatted-number. But this is not available on my Ipad (newest IOS and Chrome).
On my Ipad it looks like Dlite sees an input-field as a container.
Strange.

![image](https://user-images.githubusercontent.com/22573455/216824389-47c59484-c792-44d3-badf-5b328c2b46ac.png)

**Comment by renaudpawlak** [comment]
Well, it will be hard to understand. I don't have an IPAD. Do you think you can enable developer tools and look at the console for an error. Otherwise, I am planning to buy an IPAD anyways... I should be able to fix it then.
